docs-2.0/3.ngql-guide/8.clauses-and-options/where.md

```diff
@@ -10,7 +10,7 @@ The `WHERE` clause usually works in the following queries:
 
 ## OpenCypher compatibility
 
-* Using patterns in `WHERE` is not supported (TODO: planning), for example `WHERE (v)-->(v2)`.
+* Using patterns in expression is supported, but introducing new variables is not supported. for example, `MATCH (v:player) WHERE (v)-->()<--(v)` is supported, but `MATCH (v:player) WHERE (v)-->(t)<--(v)` is not supported. `t` is a new variables.
```

Comment:
```suggestion
* Using patterns in query statements is supported, but introducing new variables is not supported. for example, `MATCH (v:player) WHERE (v)-->()<--(v)` is supported, but `MATCH (v:player) WHERE (v)-->(t)<--(v)` is not supported. `t` is a new variables.
```

Reply:
Using `query statements` is not precise enough, the emphasis here is the pattern in the expression.